portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,34 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Akhil Binani's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Akhil Binani's Portfolio</h1>
+      <img id = "My-pic" src="image/image.jpeg" >
```

Comment:
Better to use class instead of id whenever you wish to apply css to some element. Class name should be in capital camel case - > eg : "MyPic".
Use id only when you are accessing that element in Js file.